### ISSUE TITLE
Basic usage clarification

### ISSUE THREAD

**Issue by maxfi** [Meteor-Community-Packages/ground-db#154]
When in offline mode how do I know when the collections are ready?
- `Ground.isReady()` appears to become true when a collection has successfully been grounded when online but doesn't become true when offline.
- `Ground.isResumed()` appears to become true when the grounded collection is ready, even when offline, but appears to become true prior to the collections being fully loaded with data. 

Also, could someone please clarify the following from the README:

> Example of different patterns. Grounding a Meteor.Collection will attach the cache, resume and cross tabs update offline

Basically what I'm looking to achieve is to have all my collections cached (with all records) into localStorage so that when the app is loaded when offline (utilizing appcache) it works as it would normally whilst online. Ie, the user is able to make changes to collections etc which are then then synced when the client gets an internet connection. Is there anything other than _grounding_ the collections that I need to do to get this to work?

**Comment by raix** [comment]
I've released `2.0.0-alpha.1` [Notes about the new version, migration and what it doesn't do](https://github.com/GroundMeteor/db/issues/153#issuecomment-206125703) - Theres a `loaded` event, let me know what you think

**Comment by NitroBAY** [comment]
Personally I **_hate_** 2.0.0. 
Before we just had to change Mongo to Ground.collection() and all was just wonderful, and actually gave all functionality we could expect from offline : all tabs keep syncing, data is synchronized when we go back online etc.
And now with 2.0.0 you do like if you came back at the very early state of the project (I mean I don't know how it was like at the beginning but it gives that feeling) and remove all functionalities 'hey this is just something to save data offline, now good luck about everything else', it add much code, much work, not backward compatible. I personally think that your work could fit some people who want to go more low level for optimization but this has nothing to do with wonderful and so easy to implement GroundMeteor which I am a BIG FAN. 
I really think that what you call V2 should have his own package, and maybe be a dependency of the real GroundMeteor for your usage or I don't know.
I think that I'll never use V2 and so I'm still looking for .ready() method or similar which would work both offline and online.
Please consider my proposals.
